tests/app/dao/test_services_dao.py

```diff
@@ -814,7 +814,8 @@ def test_create_service_and_history_is_transactional(notify_db_session):
     with pytest.raises(IntegrityError) as excinfo:
         dao_create_service(service, user)
 
-    assert 'column "name" violates not-null constraint' in str(excinfo.value)
+    assert 'column "name" of relation "services_history" violates not-null constraint' in str(excinfo.value)
```

Comment:
So if I understand correctly: the previous version of postgres had a slightly different error message and hence, when ran through the workflow was using that old version.. but new version has a more specific message and we updated the version to be the same locally and remotely?

Reply:
Exactly.